qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummaryReport.java

```diff
@@ -144,7 +144,7 @@ public void toXml(Element parent) {
 		
 		// create bamMertrics
 		bamReportElement = XmlElementUtils.createSubElement(bamReportElement, ProfileType.BAM.getReportName() + XmlUtils.METRICS);
-		createQNAME(XmlElementUtils.createSubElement(bamReportElement, XmlUtils.QNAME));	
+		//createQNAME(XmlElementUtils.createSubElement(bamReportElement, XmlUtils.QNAME));
```

Comment:
I would suggest removing commented out code - makes the code cleaner, and easy enough to do diffs with previous versions if you need to see what was previously there